### ISSUE TITLE
Stuck at decrypting key

### ISSUE THREAD

**Issue by fiatjaf** [mikedilger/relay-manager#5]
Whenever I run this, no matter what I do, the program halts at https://github.com/mikedilger/relay-manager/blob/6e49568228a663b5935f570790a28c4eef436243/src/signer.rs#L27 forever.

**Comment by mikedilger** [comment]
Maybe your encrypted private key uses a very high parameter for it's scrypt value?

I think the scrypt code is compiled with opt=3 even if you compile relay-manager in debug mode (based on the Cargo.toml in nostr-types), but if I'm wrong that could be an issue too.

**Comment by fiatjaf** [comment]
No, my private key doesn't take more than a second to decrypt anywhere, there must something else going on.

**Comment by mikedilger** [comment]
Does it prompt you for a password?  Do you enter a password?  Or are you running it from a script?

**Comment by fiatjaf** [comment]
Yes, it prompted me and I entered it.
